lib/routes/ui_test/ui_test_page.dart

```diff
@@ -185,6 +186,7 @@ class UITestPage extends StatelessWidget {
                     HEX.decode('0c56b71ef'),
                   ),
                   minFinalCltvExpiryDelta: 18,
+                  network: Network.Bitcoin,
```

Comment:
This change got addressed on main branch. There shouldn't be any conflicts but please rebase.

Reply:
rebased